cmd/crc/cmd/config/config.go

```diff
@@ -24,6 +24,7 @@ const (
 	HTTPSProxy           = "https-proxy"
 	NoProxy              = "no-proxy"
 	ProxyCAFile          = "proxy-ca-file"
+	ConsentTelemetry     = "consent-telemetry"
```

Comment:
not sure how well it reads in English? Should it be `telemetry-consent`, or just `enable-telemetry`?

Reply:
@gbraad ^^